lib/svtplay_dl/utils/getmedia.py

```diff
@@ -221,12 +221,23 @@ def options_subs_dl(subfixes):
         if fstream.config.get("thumbnail") and hasattr(stream, "get_thumbnail"):
             stream.get_thumbnail(stream.config)
 
-        post = postprocess(fstream, fstream.config, subfixes)
-        if fstream.audio and post.detect:
-            post.merge()
-        if fstream.audio and not post.detect and fstream.finished:
-            logging.warning("Cant find ffmpeg/avconv. audio and video is in seperate files. if you dont want this use -P hls or hds")
-        if fstream.name == "hls" or fstream.config.get("remux"):
-            post.remux()
+        if fstream.config.get("no_postprocess") is True or all(fstream.config.get(x) for x in ["no_remux", "no_merge"]) is True:
+            logging.info("All done. Not postprocessing files, leaving them completely untouched.")
+
+        if fstream.config.get("no_postprocess") is False:
```

Comment:
why do we need this? its always `False` by default.

Reply:
Yep, sorry. Forgot that one. I was too hastily thinking about the semi silent stuff.

Also, i just noticed that the semi silent log message doesn't work because `fstream.options.output` does not exist. Its pretty easy to fix this with `formatname()`. Or do you have a better solution? It's not available in fstream, so maybe it should. But that might be another PR i'm thinking.